js_modules/dagit/packages/core/src/workspace/VirtualizedGraphTable.tsx

```diff
@@ -113,11 +113,28 @@ const GraphRow = (props: GraphRowProps) => {
     <Row $height={height} $start={start}>
       <RowGrid border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}>
         <RowCell>
-          <Box flex={{direction: 'column', gap: 4}}>
-            <span style={{fontWeight: 500}}>
+          <Box flex={{direction: 'column'}}>
+            <div style={{whiteSpace: 'nowrap', fontWeight: 500}}>
```

Comment:
I wonder if we could do this with a `RowCell::first-child` CSS Rule instead of applying it to each table? (Maybe it's not something we want /that/ consistently?)

Reply:
Possibly -- I'll put this in the category of things to go back and clean up (and abstract away) once things are more settled.